param/__init__.py

```diff
@@ -1483,3 +1494,117 @@ def get_range(self):
         return abbreviate_paths(self.path,super(MultiFileSelector, self).get_range())
 
 
+class Date(Number):
+    """
+    Date parameter of datetime type.
+    """
+
+    def __init__(self, default=None, allow_None=False, **kwargs):
+        allow_None = True if default is None else allow_None
+        super(Date, self).__init__(default=default, allow_None=allow_None, **kwargs)
+        self._check_value(default)
```

Comment:
Isn't all of this the default behavior anyway?  I can't see what required there to be a constructor here, but there must be some part of this that doesn't happen already?